Foundation/include/Poco/Format.h

```diff
@@ -27,7 +27,7 @@
 namespace Poco {
 
 
-std::string Foundation_API format(const std::string& fmt, const Any& value);
+	std::string Foundation_API format(const std::string& fmt, const Any& value);
```

Comment:
why this indentation?

Reply:
this is the result of vscode auto-formatting